Assets/MRTK/Core/Inspectors/Utilities/InspectorUIUtility.cs

```diff
@@ -431,7 +431,7 @@ public static void DrawNotice(string notice)
         {
             Color prevColor = GUI.color;
 
-            GUI.color = ColorTint100;
+            GUI.color = ColorTint50;
```

Comment:
Was this intentional?

Reply:
The change was intentional.  Here is the difference between ColorTint100 and ColorTint50:

|ColorTint100| ColorTint50|
|---|---|
| ![image](https://user-images.githubusercontent.com/53493796/107270885-54316680-6a00-11eb-8f71-3c20e8e766df.png)|![image](https://user-images.githubusercontent.com/53493796/107270991-75925280-6a00-11eb-95f9-8e1d430a4274.png)|